components/sensor/xiaomi_scale.rst

```diff
@@ -0,0 +1,73 @@
+Xiaomi MiScale v1 and v2 BLE Sensor
+=========================
+
+.. seo::
+    :description: Instructions for setting up Xiaomi MiScale bluetooth-based scale in ESPHome.
+    :image: xiaomi_miscale.jpg
+    :keywords: Xiaomi, Mi Scale, Scale, BLE, Bluetooth
+
+The ``xiaomi_miscale`` sensor platform lets you track the weight output of Xiaomi MiScale V1 and V2 Bluetooth Low Energy
+devices using the :doc:`/components/esp32_ble_tracker`. This component will track the
+weight measured of the MiScale device every time the sensor
+sends out a BLE broadcast. Note that contrary to other implementations, ESPHome can track as many
+MiScale devices at once as you want.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    esp32_ble_tracker:
+
+    sensor:
+      - platform: xiaomi_miscale
+        mac_address: '94:2B:FF:5C:91:61'
+        weight:
+          name: "Xiaomi Mi Scale Weight"
+
+Configuration variables:
+------------------------
+
+- **mac_address** (**Required**, MAC Address): The MAC address of the Xiaomi MiScale device.
+- **weight** (**Required**): The information for the weight sensor.
+
+  - **name** (**Required**, string): The name for the temperature sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
```

Comment:
This is missing two entries: `impedance` and `battery_level`

Reply:
Done.

Reply:
Done.